action.yml

```diff
@@ -0,0 +1,48 @@
+name: poutine - GitHub Actions SAST
+description: |
+  BoostSecurity.io’s poutine detects vulnerabilities and misconfigurations in your GitHub Actions workflows.
+  Designed to streamline security analysis, poutine scans your repository’s CI/CD pipelines.
+  It offers insights to secure your software supply chain efficiently.
+inputs:
+  path:
+    description: 'Path to the local repository to analyze.'
+    required: true
+    default: '.'
+runs:
+  using: composite
+  steps:
+    - name: Download and Setup
+      id: setup
+      shell: bash
+      env:
+        PINNED_VERSION: v0.9.3
+        PINNED_ARCH: poutine_Linux_x86_64.tar.gz
+        PINNED_HASH: 938c36de89eea18d69c8d5605a64ef1573e682cfd02b61d4cc2de3b0663a63bc
+      run: |
+        # Create a temp dir for the tool and its results
+        POUTINE_TOOL_DIR=$(mktemp -d)
```

Comment:
this should be stored the tool cache? or the home directory at least. if you run the action twice in a workflow, the action should not have to redownload the binary.